.rubocop.yml

```diff
@@ -16,6 +23,15 @@ Style/SpaceBeforeFirstArg:
 Style/BracesAroundHashParameters:
   Enabled: false
 
+Style/CaseEquality:
+  Enabled: false
+
+Style/GuardClause:
+  Enabled: false
+
+Style/ConditionalAssignment:
+  Enabled: false
```

Comment:
Disabled these because the `if` in the middleware to set the logger was triggering it and it wasn't possible to use a guard clause.